src/sql/src/plan/func.rs

```diff
@@ -797,6 +797,13 @@ lazy_static! {
             "convert_from" => Scalar {
                 params!(Bytes, String) => BinaryFunc::ConvertFrom
             },
+            "current_schemas" => Scalar {
+                params!(Bool) => unary_op(|ecx, e| {
+                    let include_system_schemas = e.into_literal_bool().unwrap();
```

Comment:
This `unwrap` isn't safe here. Untested, but I bet `SELECT current_schemas(NOT true)` panics.

Thinking more, using `into_literal_bool` is something to avoid! Requiring literals as function arguments severely limits expressivity. The places we _do_ require literals are things like `regexp_extract` where we just _have_ to know the regexp statically.

I think here you could do something like

```rust
let with_sys = ScalarExpr::literal_array(catalog.search_path(true));
let without_sys = ScalarExpr::literal_array(catalog.search_path(false));
ScalarExpr::If {
    cond: e,
    then: with_sys,
    els: without_sys,
}
```

It'll be a tiny bit less performant, but that's ok! This doesn't need to be a particularly high-performance function.